docs/source/queries/result.md

```diff
@@ -1,6 +1,7 @@
 # Query result
```

Comment:
I think the changes in this file (and other `.md` files) should belong to the next commit.